internal/cloudapi/openapi.yml

```diff
@@ -60,6 +60,24 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ComposeResult'
+  /packages:
+    post:
```

Comment:
Should this be a `GET`?

Reply:
Doesn't allow a body and the repository descriptions are quite hard to pass as path parameters, but if you'd prefer that it would be more idiomatic I guess.